### PR TITLE
Some work

### PR DESCRIPTION
**Pull request by se7kn8**
I've renamed some classes that I found out.

### CODE REVIEW

**Review by shadowfacts — APPROVED** [review]

File: mappings/net/minecraft/util/OffsetDoubleList.mapping [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CLASS cmt net/minecraft/util/OffsetDoubleList
+	FIELD a oldList Lit/unimi/dsi/fastutil/doubles/DoubleList;
+	FIELD b offset D
+	METHOD <init> (Lit/unimi/dsi/fastutil/doubles/DoubleList;D)V
+		ARG 1 oldList
+		ARG 2 offset
+	METHOD getDouble (I)D
+		ARG 1 position
+	METHOD size ()I
```
Comment (liach) [inline mappings/net/minecraft/util/OffsetDoubleList.mapping]: Does this class implement the `java.util.Collection` interface?

Comment (kashike) [reply mappings/net/minecraft/util/OffsetDoubleList.mapping]: Yes, through the parents.

Comment (liach) [reply mappings/net/minecraft/util/OffsetDoubleList.mapping]: In yarn, we do not mark methods that cannot be obfuscated with proguard as deobfuscated. As a result, this should not be included.

Comment (Prospector) [reply mappings/net/minecraft/util/OffsetDoubleList.mapping]: oh, we're not supposed to? I've been doing that lol

Comment (liach) [reply mappings/net/minecraft/util/OffsetDoubleList.mapping]: asie told me in pr reviews before.

Comment (asiekierka) [reply mappings/net/minecraft/util/OffsetDoubleList.mapping]: Yeah. Don't.

I'll clean this up later but don't

Comment (se7kn8) [reply mappings/net/minecraft/util/OffsetDoubleList.mapping]: Ok, I removed that

File: mappings/net/minecraft/stat/ServerStatHandler.mapping [hunk 2]
```diff
@@ -0,0 +1,18 @@
+CLASS xs net/minecraft/stat/ServerStatHandler
+	FIELD b LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD c server Lnet/minecraft/server/MinecraftServer;
+	FIELD d statFile Ljava/io/File;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Ljava/io/File;)V
+		ARG 1 server
+		ARG 2 statsFile
+	METHOD a saveStats ()V
```
Comment (asiekierka) [inline mappings/net/minecraft/stat/ServerStatHandler.mapping]: If it's a ServerStatHandler, do we need the verbose *Stat/*Stats?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/stat/ServerStatHandler.mapping (hunk 2 above)
Comment (se7kn8) [reply mappings/net/minecraft/stat/ServerStatHandler.mapping]: What about "statsToString"? "toString" would conflict with the default java method.

Comment (asiekierka) [reply mappings/net/minecraft/stat/ServerStatHandler.mapping]: "asString"? Depends on the context.

Comment (liach) [reply mappings/net/minecraft/stat/ServerStatHandler.mapping]: Why to string? The return type is void here

Comment (asiekierka) [reply mappings/net/minecraft/stat/ServerStatHandler.mapping]: They're talking about another method

**Review by Prospector — APPROVED** [review]
This is a tiny PR and it looks good. We need people to approve it to get merged

**Comment by shadowfacts** [comment]
@SE7-KN8 Can you make the requested change?

**Comment by se7kn8** [comment]
Sorry, I was very busy the last days, I'll change the requested things so it can be merged.

**Pull request merged**